### DISCUSSION TITLE
When adding a remote instance, by default does it alter any of the remote configuration?

### DISCUSSION THREAD

**Discussion by mailinglists35** [Martchus/syncthingtray#289] [Q&A]
I want to connect it to a remote instance, but I see the howto is more geared towards a local initial setup. Does the app detect correctly an already configured remote instance and refrains from making any modifications to it, during setup and after connecting to it?
Thanks

**Comment by Martchus** [comment]
Syncthing Tray refrains from doing modifications to your Syncthing instance regardless whether it is local or remote. Only if you explicitly trigger an action that would cause a config change (e.g. pausing a folder) it will modify the config (again regardless whether it is local or remote).

You probably want to skip the setup wizard completely, though. It will not be able to configure connections to a remote instance. You just have to fill out the connection settings manually.

Note that the wizard will override any existing connection configuration of Syncthing Tray itself (e.g. to a remote instance) but it'll keep a backup of the overridden config as secondary config (visible in the combo box in the connection settings). This remark is only about the configuration of Syncthing Tray - as state before no config changes to your Syncthing instance are done.

**Reply by mailinglists35** [reply]
Thank you!
